models/comments.model.js

```diff
@@ -38,3 +38,9 @@ exports.insertComment = async (article_id, username, body) => {
       });
   }
 };
+
+exports.removeCommentById = (comment_id) => {
+  return db.query("DELETE FROM comments WHERE comment_id = $1 RETURNING *;", [
+    comment_id,
+  ]);
```

Comment:
Nice, but this is where you can attach a .then(...) to check for any errors that you may want to throw.